docs/apps/components/color.md

```diff
@@ -0,0 +1,190 @@
+|`this.getValues`|Get the values of all fields when values change|`Object`|
+|`this.hide`|Hides the field|`(fields: String()`|
+|`this.show`|Displays the field|`(fields: String()`|
+| `this.disable`| Disable input field from user interaction|`(fields: String(), String[])`|
+|`this.getValue`|Get A Value From a component|`(fieldName String()`|
+|`this.setData`|Set the default color|`(Value: Hexadecimal color code)`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of an alert in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+
+#### Steps to place/use the methods for components of the Page Builder
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/check1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to place/use the methods for components of the Page Builder
+          * ```
+            this.addClassName('color_zxqxhwlh', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('color_zxqxhwlh', 'abc')
+        ```
+    
+    3. `this.getValues(fieldName)`
+        ```js
+        var data = this.getValues();
+        console.log(data);
+        ```
+    
+    4. `this.hide(fields)`
+        ```js
+        var fields= ['color_zxqxhwlh']
+        this.hide(fields)
+        ```
+    
+    5. `this.show(fields)`
+        ```js
+        var fields= ['color_zxqxhwlh']
+        this.show(fields)
+        ```
+    
+    6. `this.disable(['fields'])`
+        ```js
+        this.disable(['color_zxqxhwlh'])
+        ```
+    
+    7. `this.getValue('fieldName')`
+        ```js
+        var colorname = this.getValue('color_zxqxhwlh');
+        console.log('getValue', colorname);
+        ```
+    
+    8. `this.getComponent('component_ID')`
+        ```js
+        var colorname = this.getComponent('color_zxqxhwlh');
+        console.log('getComponent', colorname);
+        ```
+    
+    9. `this.setData(Value)`
+        ```js
+        this.setData(
```

Comment:
change the setData function from multiline to single line, like this: ```js this.setData({ "component_id" : "#AEE710"  });```

Reply:
done